src/goto-programs/string_abstraction.cpp

```diff
@@ -1046,13 +1046,13 @@ void string_abstractiont::move_lhs_arithmetic(exprt &lhs, exprt &rhs)
 
 goto_programt::targett string_abstractiont::abstract_pointer_assign(
   goto_programt &dest,
-  goto_programt::targett target)
+  const goto_programt::targett target)
```

Comment:
Shouldn't this rather be `goto_programt::const_targett target` - adding the `const` to an object passed by value doesn't add much (other than forbidding the `operator++` application below)?